### DISCUSSION TITLE
The search doesn’t work at all on any plugins

### DISCUSSION THREAD

**Discussion by abbaszpro** [qbittorrent/search-plugins#236] [Q&A]
I just installed several plugins, none of them seems to work... Am I missing something?

Things I have tried: 
- I can download torrents from torrent files extensions but the search doesn't work so that means nothing is blocking access to the internet 
- I have latest version of qBittorent client (no updates are available)
- All the plugins are up to date
- Allowed access in firewall for qBittorrent so nothing is blocking it.

![Screen Shot 2023-01-09 at 6 04 25 PM](https://user-images.githubusercontent.com/18553767/211314848-ae9834b9-3c27-4c38-8b7f-e29628cb8a1d.jpg)

![Screen Shot 2023-01-09 at 6 06 19 PM](https://user-images.githubusercontent.com/18553767/211314906-1ef43a51-f59e-498d-8017-34da2bae58db.jpg)

MacOS: 12.6

Anyone have any clue what might be the caveat?

**Comment by ghost** [comment]
This is interesting... I have 4.5.0 on macOS as well. Are you sure you have the correct version of Python installed and it's in your PATH?

According to the WIKI on Plugins they are no longer using Python plugins, yet my version of QBT 4.5.0 is still using Python plugins. I had upgraded from a previous install... I am wondering if I should do a complete uninstall before installing 4.5.0 to remove all directories with QBT files.

**Comment by Melodey757** [comment]
I'm having the same problem... Search Plugins were there yesterday, gone today.. re-installing didn't work.. Everything is updated..

**Comment by HarshIceHummer** [comment]
Facing the same issue! Jackett server seems down.

**Comment by ghost** [comment]
> Facing the same issue! Jackett server seems down.

I am using Jackett right now... no issues for me. 
macOS 12.6.3
qbt 4.5.0 lt2.0
Jackett v0.20.2691

I **am** confused about what has been written in the wiki relative to what I am seeing on my system.

**Comment by abbaszpro** [comment]
> This is interesting... I have 4.5.0 on macOS as well. Are you sure you have the correct version of Python installed and it's in your PATH?
> 
> According to the WIKI on Plugins they are no longer using Python plugins, yet my version of QBT 4.5.0 is still using Python plugins. I had upgraded from a previous install... I am wondering if I should do a complete uninstall before installing 4.5.0 to remove all directories with QBT files.

Mate could you tell me which version of Python I should have (I believe I have the latest which I got from homebrew) Also how do I make sure it's in my **PATH**?

**Comment by ghost** [comment]
latest version of python works.

check version using this command in Terminal.app

`python -V`

you can check the location of python using

`which python`

you can see your PATH using

`echo $PATH`

**Comment by p060477** [comment]
> latest version of python works.
> 
> check version using this command in Terminal.app
> 
> `python -V`
> 
> you can check the location of python using
> 
> `which python`
> 
> you can see your PATH using
> 
> `echo $PATH`

in windows 10 no works at all...this is a real BUG, the same pulg ins works well if using Bit Che so the BUG is in qbittorrent...

**Comment by cencio1952** [comment]
Same problem here. Maybe I'm unable to install python (3.1) correctly (at "python -V", terminal says "command not found: python"
Any hint?

**Comment by ghost** [comment]
@cencio1952 

If command is not found and you have installed Python, then you need to add it to your PATH.
There are many tutorials online regarding this.

**Comment by ngosang** [comment]
Check you have Python 3.10 / 3.11 installed in your computer.
You can take a look at the qBittorrent log tab. You should see a line with the Python version detected:
```
21/5/23 13:18 - Python detected, executable name: 'python3', version: 3.11.3
```
